components/logins/src/lib.rs

```diff
@@ -9,21 +9,41 @@
 mod error;
 mod login;
 
-mod db;
+pub mod db;
```

Comment:
I'm mildly surprised by some of the visibility changes here - I think many of them might not need to be fully public.